include/lsst/afw/detection/Footprint.h

```diff
@@ -70,13 +70,30 @@ class Footprint : public lsst::daf::base::Citizen,
      * that to construct the Footprint.
      */
 
+    /** @brief Constructor for Footprint object
+     *
+     * @param inputSpans Shared pointer to a SpanSet defining the pixels included in
+                         the Footprint.
+     * @param region Bounding box of the MaskedImage in which the Footprint was created,
```

Comment:
Why does this refer to a MaskedImage?  This could be any type of image correct?  Same question for the other constructor.